wallet/wallet.go

```diff
@@ -0,0 +1,28 @@
+package wallet
+
+type Type string
+
+const (
+	ETH      Type = "ETH"
+	HARMONY       = "HARMONY"
+	ICON          = "ICON"
+	FLOW          = "FLOW"
+	TEZOS         = "TEZOS"
+	ZILLIQA       = "ZILLIQA"
+	POLKADOT      = "POLKADOT"
+	SOLANA        = "SOLANA"
+	AVAX          = "AVAX"
+	ALGOD         = "ALGOD"
+	COSMOS        = "COSMOS"
+	CELO          = "CELO"
+	BITCOIN       = "BITCOIN"
+	NEAR          = "NEAR"
+	HELIUM        = "HELIUM"
+	CONFLUX       = "CONFLUX"
+	TERRA         = "TERRA"
+	TAQUITO       = "TAQUITO"
+	ED            = "ED"
+	HEDERA        = "HEDERA"
+	ANY           = "ANY"
+	NONE          = "NONE"
+)
```

Comment:
This is a super nit, but I feel like the Go way of doing this would be making an Iota and then writing a mapping between those iotas and a string representation. It allows you to say that two objects with the same string rep are not equal because the underlying iota rep differs. https://yourbasic.org/golang/iota/

That being said, Iotas don't have a functional benefit here, so I think this is okay. Out of curiosity, with `queryParams["wallet_type"] = string(walletType)`, did it not pass otherwise?

Reply:
The map expects the value to be a string so I had to type cast it